src/js/shield_defs.js

```diff
@@ -3062,6 +3062,16 @@ export function loadShields(shieldImages) {
     Color.shields.white
   );
 
+  // Cyprus
+  shields["cy:national"] = {
```

Comment:
`national` is an unusual name for what seems to be a network of motorways. `cy:motorway` would be more appropriate, as would `cy:A-road` considering the `A` prefix.

It's unclear whether this tagging was invented by a Cypriot. Is there a Cyprus OSM community that we can contact about this?

Reply:
[According to Wikipedia](https://en.wikipedia.org/wiki/Roads_and_motorways_in_Cyprus), “motorway” fits the A-road network pretty well, but it does sound like B-roads and E-roads could also be part of a national numbering scheme. Maybe this was an attempt to conform to what was perceived as international terminology (not the first time I’ve seen it). The tag was introduced in [changeset 44730739](https://www.openstreetmap.org/changeset/44730739).

The community index doesn’t list any local community for Cyprus, and [the national tagging page](https://wiki.openstreetmap.org/wiki/Cyprus#Highways_in_Cyprus) doesn’t provide any guidance on route relations.

Reply:
I'm hesitant to solidify a tagging scheme that may complicate tagging guidelines for the remaining route networks in Cyprus. It's unclear how other networks in Cyprus should be tagged if we decide to support `cy:national`.

Cyprus is a small country, and the mapping community may only be a few people. Perhaps we could ask the Greek community for their perspective—certainly some of them would be familiar with Cyprus, and if there are any Cypriot mappers, there's a chance we may find them there.